packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/index.md

```diff
@@ -0,0 +1,125 @@
+
+<StackSelector snippet="prep" noSelector />
+
+## Create your integration
+
+After you have your background information, you can use the Okta Admin Console and the Application Integration Wizard (AIW) to create your SSO integration inside the Okta org associated with your developer account.
+
+>**Note:** Creating your SSO app integration doesn't automatically make it available in the [OIN](https://www.okta.com/integrations/). After you have created and tested it, you need to [submit your app integration](/docs/guides/submit-app/openidconnect/overview/) to the OIN.
+
+1. Sign in to your Okta developer account as a user with administrative privileges.
+1. In the Admin Console, go to  **Applications** > **Applications**.
+1. Click **Create App Integration**.
+
+<StackSelector snippet="create" noSelector />
+
+## Specify your integration settings
+
+This portion of the guide takes you through the steps for configuring your specific SSO integration using the Okta Admin Console.
+
+After you create your integration in the [Create your integration](#create-your-integration) step, the Admin Console opens the main settings page for your new integration. In here, you can specify **General Settings** and **Sign On** options, as well as assign the integration to users in your org. Click **Edit** if you need to change any of the options, and **Save** when you have made your changes.
+
+<StackSelector snippet="settings" noSelector />
+
+## Test your integration
+
+This portion of the guide takes you through the steps required to test your integration.
+
+### Assign users
+
+First you must assign your integration to one or more test users in your org:
+
+1. Click the **Assignments** tab.
+1. Click **Assign** and then select either **Assign to People** or **Assign to Groups**.
+1. Enter the appropriate people or groups that you want to have Single Sign-On into your application, and then click **Assign** for each.
+1. For any people that you add, verify the user-specific attributes, and then select **Save and Go Back**.
+1. Click **Done**.
+
+### Test Single Sign-On
+
+1. Sign out of your administrator account in your development org. Click **Sign out** in the upper-right corner of the Admin Console.
+1. Sign in to the Okta End-User Dashboard as the regular user that was assigned the integration.
+1. In your dashboard, click the Okta tile for the integration and confirm that the user is signed in to your application.
+
+<StackSelector snippet="test" noSelector />
+
+## Next steps
+
+* After you have completed your testing and your integration is working as expected, you can start the submission process to have your integration included in the [Okta Integration Network](https://www.okta.com/okta-integration-network/).
+* Our [Submit an app integration](/docs/guides/submit-app) guide takes you through the steps required to submit your SSO integration through the OIN Manager.
+
+## See also 
```

Comment:
Hi Chris ... should you also add these references in the See also that reside at the bottom of this page: https://developer.okta.com/docs/guides/build-sso-integration/openidconnect/before-you-begin/#prerequisites